api/src/api/v1/community_businesses/volunteer_logs.ts

```diff
@@ -398,51 +400,52 @@ const routes: Hapi.ServerRoute[] = [
       }
 
       try {
-        await knex.transaction(async (trx) =>
-          Promise.all(payload.map(async (log) => {
-            const existsInDB = log.hasOwnProperty('id');
-            const shouldUpdate = existsInDB && !log.deletedAt;
-            const shouldDelete = existsInDB && log.deletedAt;
+        const results = await Promises.some(payload.map(async (log) => {
```

Comment:
I think it would be more readable if this was all pulled out to a model method. There's a lot going on in this handler

Reply:
I agree with this, but I'd like to merge now, and then work on this immediately afterwards (just so we're not delaying deployment).

Reply:
@eliascodes can you create an issue to outline the changes proposed n I'll merge this now.